### ISSUE TITLE
add override channel to special functions

### ISSUE THREAD

**Issue by williajgr** [FrSkyRC/ETHOS-Feedback-Community#2056]
Is it possible to add override channel to special functions

**Comment by mawzthefinn** [comment]
It won’t happen as ETHOS has a significantly better implementation in the Lock function type for Free Mixes For hard-setting a channel to a given value or input and the VAR mix is better for sending arbitrary values in cases where you don’t need to lock out other mixers.

Override Channel is actively dangerous as implemented in OpenTX as it ignores Outputs limits. It’s only stuck around as OpenTX never implemented a proper replacement function. ETHOS however has done so.

**Comment by williajgr** [comment]
I extensively use overide channel.  One to set flap positions on a 3 position switch. 2 to lock the nose wheel to center when retracted. 3  To set up the 6 modes on my pix hawk drone.  Can’t amazing an easier way to do those things. One line one number.Sent from my iPhoneOn Nov 7, 2022, at 10:33 AM, Bertrand Songis @ FrSky ***@***.***> wrote:﻿
Closed #2056 as not planned.

**Comment by mawzthefinn** [comment]
For the first and third, a VAR mix is far easier and more capable. They're designed specifically for that sort of application and allows you to set all desired values from one mixer configuration page. 

For the second, a Free Mix with the Lock Function type does the same thing while also obeying any Output limits you've set.

**Comment by bsongis-frsky** [comment]
> I extensively use overide channel.  One to set flap positions on a 3 position switch. 2 to lock the nose wheel to center when retracted. 3  To set up the 6 modes on my pix hawk drone.

Even in OpenTX it's not the right way to do it. The mixer is completely bypassed and it is not nice!

**Comment by williajgr** [comment]
The var mix works great, thanks looking at the free mix for the steering lock. Not so good. Why did they take out the not b up condition?  When I made a not b up with the logical switch it wouldn’t work as the condition for the free mix.       Thanks for the help          JoeSent from my iPhoneOn Nov 8, 2022, at 4:07 AM, Bertrand Songis @ FrSky ***@***.***> wrote:﻿

I extensively use overide channel.  One to set flap positions on a 3 position switch. 2 to lock the nose wheel to center when retracted. 3  To set up the 6 modes on my pix hawk drone.

Even in OpenTX it's not the right way to do it. The mixer is completely bypassed and it is not nice!

**Comment by mawzthefinn** [comment]
They didn't take out Not B up, it's just accessed a different way.

Select B up as the Active Condition and accept. Then long-press the window and you will get a popup window with two selections, invert and edge

Select invert and you will have Not B Up

This was done as it's MUCH faster if selecting the switch by moving the physical switch.

**Comment by williajgr** [comment]
Yes that is a better way thanks      JoeSent from my iPhoneOn Nov 8, 2022, at 9:03 AM, mawzthefinn ***@***.***> wrote:﻿
They didn't take out Not B up, it's just accessed a different way.
Select B up as the Active Condition and accept. Then long-press the window and you will get a popup window with two selections, invert and edge
Select invert and you will have Not B Up
This was done as it's MUCH faster if selecting the switch by moving the physical switch.

**Comment by williajgr** [comment]
I haven’t figured out how to get the individual cell voltages on my lipo sensor.   ThanksSent from my iPhoneOn Nov 8, 2022, at 9:21 AM, George Williams ***@***.***> wrote:﻿Yes that is a better way thanks      JoeSent from my iPhoneOn Nov 8, 2022, at 9:03 AM, mawzthefinn ***@***.***> wrote:﻿
They didn't take out Not B up, it's just accessed a different way.
Select B up as the Active Condition and accept. Then long-press the window and you will get a popup window with two selections, invert and edge
Select invert and you will have Not B Up
This was done as it's MUCH faster if selecting the switch by moving the physical switch.

**Comment by mawzthefinn** [comment]
Long press on the sensor once selected in the widget and you will get all the sub values available from that sensor.

It's worth long-pressing any field that shows the 3 bars in the upper left corner if you are looking for extra options.

**Comment by williajgr** [comment]
Thanks, worked like a charm.  I was trying the long press in the telemetry screen.  Really appreciate the help.  I have 2 more things I ran across today.  1. How do I get at the telemetry sent down by my pixhawk?  I see it all in the yaapu screen so I know it’s there somewhere.  I use the altitude to put down my retracts as I landed it once with them up.  2 is there a function like bgmusic, I like to have a switch that plays Top Gun on the radio.  When I tried play track it never goes off until I turn the radio off.   Thanks again          JoeSent from my iPhoneOn Nov 9, 2022, at 1:12 PM, mawzthefinn ***@***.***> wrote:﻿
Long press on the sensor once selected in the widget and you will get all the sub values available from that sensor.
It's worth long-pressing any field that shows the 3 bars in the upper left corner if you are looking for extra options.

**Comment by williajgr** [comment]
Any chance they would add half expo to stock curves for throttle?Sent from my iPhoneOn Nov 9, 2022, at 6:25 PM, George Williams ***@***.***> wrote:﻿Thanks, worked like a charm.  I was trying the long press in the telemetry screen.  Really appreciate the help.  I have 2 more things I ran across today.  1. How do I get at the telemetry sent down by my pixhawk?  I see it all in the yaapu screen so I know it’s there somewhere.  I use the altitude to put down my retracts as I landed it once with them up.  2 is there a function like bgmusic, I like to have a switch that plays Top Gun on the radio.  When I tried play track it never goes off until I turn the radio off.   Thanks again          JoeSent from my iPhoneOn Nov 9, 2022, at 1:12 PM, mawzthefinn ***@***.***> wrote:﻿
Long press on the sensor once selected in the widget and you will get all the sub values available from that sensor.
It's worth long-pressing any field that shows the 3 bars in the upper left corner if you are looking for extra options.
